test/src/server/routes/type-editor/relationship-type.js

```diff
@@ -0,0 +1,188 @@
+import {createEditor, createRelationshipType, truncateEntities} from '../../../../test-helpers/create-entities';
+import app from '../../../../../src/server/app';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import orm from '../../../../bookbrainz-data';
+
+
+const {RelationshipType} = orm;
+
+chai.use(chaiHttp);
+const {expect} = chai;
+
+describe('Relationship Type routes with Relationship Editor priv', () => {
+	const id1 = 1;
+	let agent;
+	before(async () => {
+		try {
+			await createRelationshipType(id1);
+			await createEditor(123456, 4);
+		}
+		catch (error) {
+			// console.log(error);
+		}
+		// Log in; use agent to use logged in session
+		agent = await chai.request.agent(app);
+		await agent.get('/cb');
+	});
+	after(truncateEntities);
+
+	it('should not throw an error if creating new relationship type', async () => {
+		const res = await agent.get('/relationship-type/create');
+		expect(res.ok).to.be.true;
+		expect(res).to.have.status(200);
+	});
+
+	it('should not throw an error trying to edit an existing relationship type', async () => {
+		const relationshipTypeId = 1;
+		const res = await agent.get(`/relationship-type/${relationshipTypeId}/edit`);
+		expect(res.ok).to.be.true;
+		expect(res).to.have.status(200);
+	});
+
+	it('should throw a 404 error when trying to edit a non-existent relationship type', async () => {
+		const relationshipTypeId = 222;
+		const res = await agent.get(`relationship-type/${relationshipTypeId}/edit`);
```

Comment:
Found the issue ! :D 
Sneaky one that one.
```suggestion
		const res = await agent.get(`/relationship-type/${relationshipTypeId}/edit`);
```